### PR TITLE
Upgrading Echopype and Adjust Processing

### PR DESCRIPTION
**Pull request by cwingard**
Updates the code to use echopype 0.6.3 and adjusts the processing code to account for needed changes after the update.

### CODE REVIEW

File: ooi_zpls_echograms/zpls_echogram.py [hunk 1]
```diff
@@ -72,43 +84,53 @@
...
         'instrument_orientation': 'up'
     },
     'CP03ISSM': {
         'long_name': 'Coastal Pioneer, Inshore Surface Mooring',
         'tilt_correction': 15,
         'colorbar_range': [-90, -50],
-        'vertical_range': [0, 100],
-        'deployed_depth': 95,
+        'vertical_range': [0, 95],
+        'deployed_depth': 92,
```
Comment (snwhite) [inline ooi_zpls_echograms/zpls_echogram.py:108]: The design depth is 95 m (and we averaged 93.53).

**Review by snwhite — COMMENTED** [review]
I am not an expert in echopype or python, but I did review all files.  I had one minor comment on CP03ISSM deploy depth.  Otherwise, it seems okay to my eyes.

File: ooi_zpls_echograms/zpls_echogram.py (hunk 1 above)
Comment (cwingard) [reply ooi_zpls_echograms/zpls_echogram.py]: Updated the deployed the depth

**Review by asmith-whoi — APPROVED** [review]
All files reviewed, changes look good

**Pull request merged**